vue-frontend/src/portfolio-json/togness-data.js

```diff
@@ -164,7 +164,7 @@ export default {
           {
             aspectRatio: "164%",
             alt: "togness-apply-filters-slide-video-screen",
-            title: "togness",
+            title: "",
```

Comment:
This title will decide css of video, See template code.
Need to find a way to hide it from the UI